spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/service/TaskExecutionService.java

```diff
@@ -94,4 +94,14 @@ public interface TaskExecutionService {
 	 * @since 2.8
 	 */
 	Integer getAllTaskExecutionsCount(boolean onlyCompleted, String taskName);
+
+	/**
+	 * Returns the count of all the task execution IDs with the option to include only the completed task executions.
+	 * @param onlyCompleted filter by completed task executions
+	 * @param taskName the task name, if null then retrieve all the tasks
+	 * @param days the days before the task should ended
+	 * @return the number of executions
```

Comment:
I know the other similar method was returning big integer, but I am pretty sure in the "no data" case it returns 0. I would just state that fact in the `@return` something like "never null, 0 if no data" etc..